Frameworks/UIKit/StarboardXaml/LayerCoordinator.cpp

```diff
@@ -765,12 +774,12 @@ void LayerCoordinator::_ApplyContentCenter(FrameworkElement^ element, Rect conte
         bottom = std::max<int>(0, bottom);
 
         // Cap the left/right to the maximum width
-        int maxWidth = static_cast<int>(contentSize.Width / 3);
+        int maxWidth = static_cast<int>(imageWidth / 3);
```

Comment:
This caps left/right/top/bottom portions of the NineGrid to not exceed the size of the center - otherwise the Xaml Image won't render at all.

---
In reply to: [92279685](https://github.com/Microsoft/WinObjC/pull/1530#discussion_r92279685) [](ancestors = 92279685)